CHANGELOG.md

```diff
@@ -1,208 +1,295 @@
-- INSTALL to 'bin' subfolder
+
+### Additions
+
+* CUDA 12.1.0 support
+* cuDNN 8.8.1 support
+* LLVM 16.0.0 support
+* New options:
+  * `--default-preprocessor` (synonymous with `--skip-excluded-preprocessor-conditional-blocks`)
+  * `--no-undocumented-features`
+  * `--no-warnings-on-undocumented-features`
+  * `--versions`
+
+### Fixes
+
+* Accessing `half2 struct` members (undocumented feature)
+* Moved `INSTALL` to the `bin` subfolder
 
 ## HIPIFY for ROCm 5.5.0
-### Added
-- Partial CUDA 12.0.0 support
-- cuDNN 8.7.0 support
-- Initial MIOpem support
-- cuBLAS 64bit API (CUDA 12.0) initial support
-- rocBLAS and MIOpen synthetic tests
-- LLVM 15.0.7 support
-### Misc
-- Synthetic unit tests for cuBLAS2rocBLAS and cuDNN2MIOpen
+
+### Additions
+
+* Partial CUDA 12.0.0 support
+* cuDNN 8.7.0 support
+* Initial MIOpem support
+* cuBLAS 64-bit API (CUDA 12.0) initial support
+* rocBLAS and MIOpen synthetic tests
+* LLVM 15.0.7 support
+
+### Changes
+
+* Synthetic unit tests for `cuBLAS2rocBLAS` and `cuDNN2MIOpen`
 
 ## HIPIFY for ROCm 5.4.1
-### Added
-- CUDA 11.8.0 support
-- cuDNN 8.6.0 support
-- Device types support
-- LLVM 15.0.4 support
-### Fixed
-- [Linux] Get rid of any RPATH definitions
+
+### Additions
+
+* CUDA 11.8.0 support
+* cuDNN 8.6.0 support
+* Device types support
+* LLVM 15.0.4 support
+
+### Fixes
+
+* Removed `RPATH` definitions (Linux)
 
 ## HIPIFY for ROCm 5.4.0
-### Added
-- hipRTC support
-- Error Handling API support
-- hipBLAS synthetic tests
-- LLVM 15.0.0 support
+
+### Additions
+
+* hipRTC support
+* Error handling API support
+* hipBLAS synthetic tests
+* LLVM 15.0.0 support
 
 ## HIPIFY for ROCm 5.3.0
-### Added
-- CUDA 11.7.0 support
-- cuDNN 8.4.1 support
-- CUB initial support
-- More synthetic tests
-- New options:
-  - --hip-kernel-execution-syntax
-### Fixed
-- Patches for LLVM 14.0.x (Windows only)
-- Add GNUInstallDirs for CMake on Linux
-### Misc
-- LLVM 3.8.0 is out of support
-- HIPIFY-specific options support in unit testing
+
+### Additions
+
+* CUDA 11.7.0 support
+* cuDNN 8.4.1 support
+* CUB initial support
+* More synthetic tests
+* New options:
+  * `--hip-kernel-execution-syntax`
+
+### Fixes
+
+* Patches for LLVM 14.0.x (Windows only)
+* Add` GNUInstallDirs` for CMake on Linux
+
+### Changes
+
+* LLVM 3.8.0 is out of support
+* HIPIFY-specific options support in unit testing
 
 ## HIPIFY for ROCm 5.2.0
-### Added
-- CUDA 11.6.0 support
-- cuDNN 8.3.3 support
-- LLVM 14.0.0 support
+
+### Additions
+
+* CUDA 11.6.0 support
+* cuDNN 8.3.3 support
+* LLVM 14.0.0 support
 
 ## HIPIFY for ROCm 5.1.0
-### Added
-- CUDA 11.5.1 support
-- cuDNN 8.3.2 support
-### Fixed
-- hipification of cuOccupancyMaxPotentialBlockSize and cuOccupancyMaxPotentialBlockSizeWithFlags
+
+### Additions
+
+* CUDA 11.5.1 support
+* cuDNN 8.3.2 support
+
+### Fixes
+
+* Hipification of `cuOccupancyMaxPotentialBlockSize` and
+  `cuOccupancyMaxPotentialBlockSizeWithFlags`
```

Comment:
Probably, an excessive line break

Reply:
I'm keeping the lines at 80 characters or less.